src/Message/BasicMessageInterface.php

```diff
@@ -116,23 +116,23 @@ public function getInReplyTo(): array;
     public function getReferences(): array;
 
     /**
-     * Get first body HTML part.
+     * Get message parts by type.
      *
-     * @return null|string Null if message has no HTML message part
+     * @return string[]
      */
-    public function getBodyHtml(): ?string;
+    public function getAllContentsByType(string $type): array;
 
     /**
-     * Get all body HTML parts as array.
+     * Get first body HTML part.
      *
-     * @return string[]
+     * @return null|string Null if message has no HTML message part
      */
-    public function getBodyHtmlParts(): array;
```

Comment:
Hi, I won't release a new MAJOR just for this fix, so please don't add BC-Breaks

Reply:
@Slamdunk thank for the feedback, good point, I restored the function